### DISCUSSION TITLE
Issue regarding 'Syntax Highlight' not working for Doxygen in coc-hover for function definitions.

### DISCUSSION THREAD

**Discussion by ampandey-1995** [neoclide/coc.nvim#5207] [General]
Below is the screen shot.
![image](https://github.com/user-attachments/assets/678b5efb-ce4e-453e-a744-2212538857c0)

**Comment by ampandey-1995** [comment]
@chemzqm  any ways to support highlighting syntax in coc-hover function document display?

**Comment by ampandey-1995** [comment]
Hi @fannheyward , any ways/workaround to enable the doxygen syntax highlight support in coc-hover (definitionHover) for a function?

**Reply by fannheyward** [reply]
coc.nvim can only highlight markdown documents on coc-hover.

**Comment by chemzqm** [comment]
Need to know what returned from the server, see https://github.com/neoclide/coc.nvim/wiki/Debug-language-server#using-output-channel
